mil_common/utils/mil_tools/mil_ros_tools/extract_bag_images.py

```diff
@@ -197,7 +197,7 @@ def _save_img(self, msg, time, image_dir, prefix=""):
         if self.encoding == ImageProc.COLOR or self.encoding == ImageProc.RECT_COLOR:
             img = cvtColor2(img, self.image_set.color_encoding, "bgr8")
 
-        img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
+        # img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
```

Comment:
Weird to leave this as commented out, we should add either add an explanation comment or just remove this line.